### DISCUSSION TITLE
Drop All Database Foreign Key When Upgrade to prisma 3.1.1 !!

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#9581] [Q&A] [answered]
steps:
1- upgrade from prisma 2.30.3 to 3.1.1
2- run npx prisma format
3- npx prisma migrate 
i did not make any change to my schema and found that a new migration file that contains scripts remove all foreign keys and recreated them again with a different names
how can i avoid this behavior .. ??!
please help
Thanks

**Comment by ryands17** [comment, marked as answer]
@BishoyNarouz 👋 

Follow [this](https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3) guide to upgrade from Prisma 2 to Prisma 3 and let me know if that works.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
